Wikipedia/Code/TalkPageCellTopicView.swift

```diff
@@ -179,9 +192,15 @@ final class TalkPageCellTopicView: SetupView {
 extension TalkPageCellTopicView: Themeable {
 
     func apply(theme: Theme) {
-        topicLabel.textColor = theme.colors.primaryText
+        
+        topicTitleTextView.attributedText = viewModel?.topicTitle.byAttributingHTML(with: .headline, boldWeight: .semibold, matching: traitCollection, color: theme.colors.primaryText, linkColor: theme.colors.link, handlingLists: false, handlingSuperSubscripts: true)
+        topicTitleTextView.backgroundColor = theme.colors.paperBackground
+        
         timestampLabel.textColor = theme.colors.secondaryText
-        topicCommentLabel.textColor = theme.colors.primaryText
+        
+        let commentColor = (viewModel?.isThreadExpanded ?? false) ? theme.colors.primaryText : theme.colors.secondaryText
```

Comment:
Just a personal note, I understand it's as designed but it's kind of visually unexpected to see this change as the expand/collapse is toggled.